content/intro-to-storybook/react/en/simple-component.md

```diff
@@ -46,15 +46,12 @@ Below we build out Task’s three test states in the story file:
 // src/components/Task.stories.js
 
 import React from 'react';
-import { action } from '@storybook/addon-actions';
 
 import Task from './Task';
 
-export default {
-  component: Task,
-  title: 'Task',
-  // Our exports that end in "Data" are not stories.
-  excludeStories: /.*Data$/,
+export const actionsData = {
+  onPinTask: { action: 'onPinTask' },
+  onArchiveTask: { action: 'onArchiveTask' },
 };
```

Comment:
Can we use argtypes for the actions?

I'm not sure what's better, either we could:
 - Define a global `argTypeRegex` in `preview.js` or
 - Add action annotations per argTypes in each component.

WDYT @jonniebigodes ?

Reply:
Awesome!